Specs/Number/Swedish/NumberModel.json

```diff
@@ -1376,30 +1358,30 @@
           "value": "0,0476190476190476"
         },
         "Start": 0,
-        "End": 17
+        "End": 16
       }
     ]
   },
   {
-    "Input": "en tjugo femtedel",
```

Comment:
"en tjugo femtedel" is not a valid case?

Reply:
This has been fixed